ssg/build_yaml.py

```diff
@@ -130,6 +130,7 @@ def from_yaml(cls, yaml_file, env_yaml=None):
         if selection_entries:
             profile._parse_selections(selection_entries)
         del yaml_contents["selections"]
+        policies = yaml_contents.pop("policies", None)
```

Comment:
I think that we could add a method `resolve_control` that would assume that we *somehow* got a policy-control pair, and it would translate it to selections, values and refinements using the manager.
I would love to have implementation details about how to get policies and controls in an another PR.